src/cards/ha-history_graph-card.html

```diff
@@ -0,0 +1,91 @@
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
+<link rel="import" href="../../bower_components/paper-card/paper-card.html">
+
+<link rel="import" href="../components/state-history-charts.html">
+<link rel="import" href="../data/ha-state-history-data.html">
+<link rel='import' href='../util/hass-mixins.html'>
+
+<dom-module id='ha-history_graph-card'>
+  <template>
+    <style>
+      .content {
+        padding: 0 16px 16px;
+      }
+      paper-card {
+        width: 100%;
+      }
+    </style>
+    <ha-state-history-data
+      hass='[[hass]]'
+      filter-type='recent-entity'
+      entity-id='[[computeHistoryEntities(stateObj)]]'
+      data='{{stateHistory}}'
+      is-loading='{{stateHistoryLoading}}'
+      cache-config='[[computeCacheConfig(stateObj)]]'
+    ></ha-state-history-data>
+    <paper-card
```

Comment:
Oh snap, I realized now that this might not make this component look alike with the other cards that we show. Can you post a screenshot of this component together with something like the demo light platform? The headers/content should be aligned 

Also, I'm confused to see the references to `inDialog` in this component. They should not be here. If the more info dialog needs things from this component, we should extract it out. I don't want more info dialog and cards to get coupled as I'm thinking about code splitting the more info dialogs out in the future.

Reply:
I switched to manual header like in `ha-entities-card` instead of paper-card styled one.

Reply:
Regarding inDialog:

The same UI is used as a card and as a more-info of that card (with the exception of some padding removal)